multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +200,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /etc/rkthack
```

Comment:
I'll change it to `/opt/bin` and `rkt-hostns-wrapper` seems fine to me too. 

The reason I call it out as a hack is because of how we configure this solution rather then the solution itself. I think escaping the chroot is generally the right thing to do when a fly container wants to call host binaries, but i'd like to see us actually support this as some sort of stage1-fly feature flag or something rather then manually adding a bash script to the host specifically for rkt. 

I thought I had made an issue about removing this in the future, but I can't find it now. The relevant rkt issue is: https://github.com/coreos/rkt/issues/2878